### ISSUE TITLE
a11y: Project management history table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6078]
Minor accessibility improvements can be made to `/manage/project/{project_name}/history/`:

* The history `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node
